### ISSUE TITLE
[BUG] RM on untracked pds member doesn't remove omvs copy

### ISSUE THREAD

**Issue by mfsysprog** [wizardofzos/zigi#372]
**Describe the bug**
RM on an untracked pds member will not remove omvs copy

**Release of zigi**
2.9

**To Reproduce**
1. RM an untracked member of a pds
2. Status will show *Removed
3. PF3 out 
4. Go back in with S and the member is still there (at least the omvs copy is)

**Expected behavior**
WWRF_handle_lcmd_rm has following statement:
cmd = 'cd' xldir '&& git rm -f "'usssafe(m)'"' 
x = docmd(cmd,,so.,se.) 

When doing a git rm on an untracked file it will result in:
fatal: pathspec '<untacked_membername>' did not match any files
since git doesn't know about the file, because it is not tracking it. 

Perhaps best fix would be to trap this message and subsequently do a normal rm?

**Screenshots**

**Additional context**

**Comment by lbdyck** [comment]
Good catch - this is easily fixed.

**Comment by lbdyck** [comment]
In investigating this issue the RM also fails with file extensions.

**Comment by mfsysprog** [comment]
elegant fix!

**Comment by lbdyck** [comment]
Glad you like it 😄

**Comment by mfsysprog** [comment]
I actually decided to go and ask the git mailinglist if it would be feasible to add a -u option to the git rm command to remove untracked files.
https://lore.kernel.org/git/1983783029.127560.1590607913242@mail.ziggo.nl/T/#t
I learned about both the git clean option to remove untracked files and the option to set aliasses.
Did a little test to set the following alias:
git config --global alias.rmclean '!git rm -r -f $1 2>/dev/null; git clean -f $1 2>/dev/null'
this allows you to do a git rmclean command that will execute both a git rm and a git clean in one go.
I will probably get some remarks about it, I haven't really given it enough thought if there are no side effects, but I thought I should mention it for the principle. Especially the alias option might come in handy somewhere.
